test/testcafe/spec/UnlockAccountChallenge_spec.js

```diff
@@ -84,9 +84,11 @@ test.requestHooks(identifyLockedUserMock)('should show unlock account authentica
 
   const successPage = new TerminalPageObject(t);
   await t.expect(successPage.getFormTitle()).eql('Account successfully unlocked!');
-  await t.expect(successPage.getMessages()).eql('You can log in using your existing username and password.');
-  const gobackLinkExists = await successPage.goBackLinkExists();
-  await t.expect(gobackLinkExists).eql(false);
+  await t.expect(successPage.doesTextExist('You can log in using your existing username and password.')).eql(true);
+  if(!userVariables.v3) {
+    const gobackLinkExists = await successPage.goBackLinkExists();
+    await t.expect(gobackLinkExists).eql(false);
+  }
```

Comment:
made this conditional since goBackLink and signoutLink are the same in v3

Reply:
Is this conditional something we could put into the `TerminalPageObject` instead?

Reply:
I dont think so since it will probably break other tests that only test for goBackLink existence or only test for signoutLink existence.